src/utils.rs

```diff
@@ -39,6 +39,15 @@ pub fn print_warning<S: AsRef<str>>(warning: S) {
     }
 }
 
+/// Prints a warning to `stderr` in yellow.
```

Comment:
On develop, when one of the fields of the config files where not properly filled, it raised a warning, but kept the well formed values.

With the current code, Serde stops deserializing when some field is incorrect and returns an Error. This method is to print this error that serde is returning. Now, in this case, we are ignoring ALL the configuration loaded from the malformed file and keeps loading the configuration from other sources (CLI, for example).

If you want, I can bubble up the serde error if there is some malformed value and we can remove this method.